tfhe/src/shortint/server_key/tests/shortint.rs

```diff
@@ -4,9 +4,29 @@ use paste::paste;
 use rand::Rng;
 
 /// Number of assert in randomized tests
-const NB_TEST: usize = 30;
+#[cfg(not(feature = "coverage"))]
+const NB_TEST: usize = 200;
```

Comment:
may be worth a try is

```rust
const NB_TEST: usize = {
    #[cfg(not(feature = "__coverage"))]
    200
    #[cfg(feature = "__coverage")]
    2
};
```

legal rust ? (to keep things locally)

also use the private feature

I don't remember if I already asked why 200 ? that seems like a lot ? Or is it linked to getting the loop out of the default tests and adding separate dirty inputs test ?

Reply:
Yeah 200 seems a lot. I'll switch back to 30 and we'll use that values for the to be implemented dirty inputs tests.

Reply:
Sadly, you're proposal is not legal Rust.

Reply:
😭